### PR TITLE
mes-1738 Change candidate details to modal

### PR DESCRIPTION
**Pull request by Jason-Pullen**
## Description and relevant Jira numbers
mes-1738
Changed the candidate details to launch modal, added Close with X Icon to the header. Removed the logout header button

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

Formatted and scaling to accessibility settings 
screenshot 2019-03-07 at 08 35 07
screenshot 2019-03-07 at 08 35 37

### CODE REVIEW

File: src/app/app.scss [hunk 1]
```diff
@@ -34,4 +34,14 @@ img {
   -webkit-user-drag: none;
   -webkit-user-select: none;
   -ms-user-select: none;
+}
+.modal-fullscreen {
```
Comment (ammarhaiderbjss) [inline src/app/app.scss]: Could we put this in a separate file and import it in?

File: src/pages/journal/components/candidate-link/candidate-link.ts [hunk 2]
```diff
@@ -22,10 +21,14 @@ export class CandidateLinkComponent {
   @Input()
   isPortrait: boolean;
 
-  constructor(public navController: NavController) {
+  constructor(public modalController: ModalController) {
   }
 
-  navigateToCandidateDetails() {
-    this.navController.push('CandidateDetailsPage', { slotId: this.slotId, slotChanged: this.slotChanged });
+  openCandidateDetailsModal() {
```
Comment (ammarhaiderbjss) [inline src/pages/journal/components/candidate-link/candidate-link.ts]: quick test around this?

File: src/pages/journal/components/candidate-link/__tests__/candidate-link.spec.ts [hunk 3]
```diff
@@ -43,12 +43,13 @@ describe('CandidateLinkComponent', () => {
       expect(component).toBeDefined();
     });
 
-    it('should call the push function of navController and pass the right slotId', () => {
-      component.navigateToCandidateDetails();
+    it('should call the create function of navController and pass the right slotId', () => {
```
Comment (johnmorleybjss) [inline src/pages/journal/components/candidate-link/__tests__/candidate-link.spec.ts]: The name of this test still says `navController`.

Comment (Jason-Pullen) [reply src/pages/journal/components/candidate-link/__tests__/candidate-link.spec.ts]: changed

File: src/pages/journal/components/candidate-link/candidate-link.ts (hunk 2 above)
Comment (Jason-Pullen) [reply src/pages/journal/components/candidate-link/candidate-link.ts]: there is a test in there for the openCandidateDetailsModal

**Pull request merged**